docs/installation.rst

```diff
@@ -1,244 +1,15 @@
-
-.. code:: bash
-
-   $ npm uninstall -g aws-sam-local
-
-Advanced installations
-----------------------
-
-Build From Source
-^^^^^^^^^^^^^^^^^
-
-First, install Python(2.7 or 3.6) on your machine, then run the following:
-
-.. code:: bash
-
-   # Clone the repository
-   $ git clone git@github.com:awslabs/aws-sam-cli.git
-
-   # cd into the git
-   $ cd aws-sam-cli
-
-   # pip install the repository
-   $ pip install --user -e .
-
-Install with PyEnv
-^^^^^^^^^^^^^^^^^^
-
-.. code:: bash
-
-    # Install PyEnv (https://github.com/pyenv/pyenv#installation)
-    $ brew update
-    $ brew install pyenv
-
-    # Initialize pyenv using bash_profile
-    $ echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi\nexport PATH="~/.pyenv/bin:$PATH"' >> ~/.bash_profile
-    # or using zshrc
-    $ echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi\nexport PATH="~/.pyenv/bin:$PATH"' >> ~/.zshrc
-
-    # restart the shell
-    $ exec "$SHELL"
-
-    # Install Python 2.7
-    $ pyenv install 2.7.14
-    $ pyenv local 2.7.14
-
-    # Install the CLI
-    $ pip install --user aws-sam-cli
-
-    # Verify your installation worked
-    $ sam –-version
-
-Updating SAM CLI on AWS Cloud9
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-If your AWS Cloud9 environment has a SAM CLI version < 0.3.0 installed there are a few extra steps you must do to upgrade to newer versions:
-
-.. code:: bash
-
-    # Uninstall the older version of SAM Local
-    $ npm uninstall -g aws-sam-local
-   
-    # Remove the symlink 
-    $ rm -rf $(which sam)
-   
-    # Install the CLI
-    $ pip install --user aws-sam-cli
-   
-    # Create new symlink
-    $ ln -sf $(which sam) ~/.c9/bin/sam
-    
-    # Reset the bash cache
-    $ hash -r
-   
-    # Verify your installation worked
-    $ sam –-version
-
-Troubleshooting
----------------
-
-Mac Issues
-^^^^^^^^^^
-
-1. **TLSV1_ALERT_PROTOCOL_VERSION**:
-
-If you get an error something similar to:
-
-::
-
-   Could not fetch URL https://pypi.python.org/simple/click/: There was a problem confirming the ssl certificate: [SSL: TLSV1_ALERT_PROTOCOL_VERSION] tlsv1 alert protocol version (_ssl.c:590) - skipping
-
-then you are probably using the default version of Python that came with
-your Mac. This is outdated. So make sure you install Python again using
-homebrew and try again:
-
-.. code:: bash
-
-   $ brew install python
-
-Once installed then repeat the `Installation process <#windows-linux-macos-with-pip>`_
-
-Learn More
-==========
-
--  `Project Overview <../README.rst>`__
--  `Getting started with SAM and the SAM CLI <getting_started.rst>`__
--  `Usage <usage.rst>`__
--  `Packaging and deploying your application <deploying_serverless_applications.rst>`__
--  `Advanced <advanced_usage.rst>`__
+==============
+Installation
+==============
+
+Installation Instructions for AWS SAM CLI are linked below.
+
+-  `Pip`_
```

Comment:
Can we put Pip last? I think we want customers to use Pip as a last effort and use the other options first.

Reply:
Yes, thats a good suggestion!